gtk4/src/auto/media_stream.rs

```diff
@@ -26,9 +26,6 @@ pub trait MediaStreamExt: 'static {
     #[doc(alias = "gtk_media_stream_ended")]
     fn ended(&self);
 
-    //#[doc(alias = "gtk_media_stream_error")]
-    //fn error(&self, domain: glib::Quark, code: i32, format: &str, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs);
```

Comment:
You probably want to implement the `gtk_media_stream_gerror` variant though. But who is even supposed to call this function?

Reply:
This is called by custom MediaStream/MediaFile implementations to set the media in error state. This can be retrieved by widgets making use of it to display the error. I suppose, we should only go with the `gerror` variant? Not sure if the `gtk_media_stream_error` variant is any useful.

Reply:
I ended up removing the `error` variant & renaming the `gerror` to `error`

Reply:
That makes most sense, yes